rust/signed_doc/src/doc_types/mod.rs

```diff
@@ -53,3 +57,37 @@ pub const IMMUTABLE_LEDGER_BLOCK_UUID_TYPE: Uuid =
     Uuid::from_u128(0xD9E7_E6CE_2401_4D7D_9492_F4F7_C642_41C3);
 /// Submission Action `UuidV4` type.
 pub const SUBMISSION_ACTION: Uuid = Uuid::from_u128(0x7892_7329_CFD9_4EA1_9C71_0E01_9B12_6A65);
+
+// -------- Mapping old document types to new document types --------
```

Comment:
Move these definitions on the top of the file, and there are not mappings from old one to the new one, there are document type definitions.
Also all these doc types should be relied on non `deprecated` consts